Makefile

```diff
@@ -1,6 +1,5 @@
-
-# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-# See LICENSE for license information.
+# Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
```

Comment:
@phoinixgrr would you know?

Reply:
This was mostly inspired by: 
- https://github.com/opencord/voltha-openonu-adapter-go/blob/master/Makefile
- https://github.com/edgexfoundry/edgex-go/blob/main/Makefile

End goal, is for the Release team, to automate maintaining all Makefiles for all projects, as part of release templating, and be CODEOWNERS 

But not strongly opinionated on this.

Reply:
It is ok to remove the the licence header since we have LICENCE file at the repo. And it is nice to follow Apache recommendations.

```
It is also valuable to tag each of your source-code files in case they become detached from the LICENSE file. To apply the Apache License to your source-code files, one approach is to attach the following notice to as a comment at the top of each file. 
```
Please see https://www.apache.org/foundation/license-faq.html#My-License for more details. Section: HOW SHOULD I APPLY THE APACHE LICENSE TO MY OWN SOFTWARE?